wandb/proto/wandb_internal.proto

```diff
@@ -479,6 +479,9 @@ message AlertResult {
  * Request: all non persistent messages
  */
 message Request {
+  reserved 14, 15, 16;
+  reserved "ArtifactSendRequest", "ArtifactPollRequest", "ArtifactDoneRequest";
+
```

Comment:
reserved isnt needed. thanks for the diligence though.
these are not persistent
```suggestion
```